uavcan/equipment/esc/1036.StatusExtended.uavcan

```diff
@@ -0,0 +1,16 @@
+#
+# Extended Generic ESC status.
+# Additional fields on top of the generic ESC status message.
+# Transmitted at a lower rate.
+#
+
+uint7 input_pct               # Input command to ESC, in percent, which is commanded using the setpoint messages. Range 0% to 100%.
+uint7 output_pct              # Output command from ESC to motor, in percent. Range 0% to 100%. 
+
+int9 motor_temperature_degC   # Temperature of connected motor, in Celsius. Range is -256 to +255 C.
+uint9 motor_angle             # Measured angle of connected angle sensor, in degrees. Range is 0 to 360. 
+
+uint24 status_flags           # Manufacturer-specific status flags currently active. 
```

Comment:
we can make this uint32 again now, packet is:
7+7+9+9+24 == 56
we can have 64 bits in one frame, so might as well make status uint32 and it will be exactly one frame